library/src/main/java/com/github/gfx/android/orma/Selector.java

```diff
@@ -225,12 +225,22 @@ public boolean isEmpty() {
 
     @Nullable
     public Model valueOrNull() {
+        final long localOffset = offset;
+        if (localOffset > 0) {
+            return getOrNull(localOffset);
+        }
         return getOrNull(0);
```

Comment:
I think it is `getOrNull()` that should respect `offset`.

BTW I preffer `Math.max(offset, 0)` because it is simpler than if statements.